.github/workflows/sync_to_airtable.yml

```diff
@@ -2,59 +2,292 @@ name: Sync to Airtable
+                number: context.payload.issue.number.toString(),
+                secret: process.env.SECRET,
+                title: context.payload.issue.title,
+              });
+              return fetch(process.env.WEBHOOK, {
+                body,
+                headers: {
+                  "Content-Type": "application/json"
+                },
+                method: "POST"
+              });
+  issue_reopened:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'reopened'
+    steps:
+        - name: Set Data
+          id: set-reopened-data
+          uses: actions/github-script@v6
+          env:
+            SECRET: ${{ secrets.AIRTABLE_KEY }}
+            WEBHOOK: ${{ secrets.AIRTABLE_WEBHOOK }}
+          with:
+            script: |
+              const body = JSON.stringify({
+                action: context.payload.action,
+                data: JSON.stringify(context.payload.issue),
+                owner: context.payload.repository.owner.login,
+                repo: context.payload.repository.name,
+                number: context.payload.issue.number.toString(),
+                secret: process.env.SECRET,
+                title: context.payload.issue.title,
+              });
+              return fetch(process.env.WEBHOOK, {
+                body,
+                headers: {
+                  "Content-Type": "application/json"
+                },
+                method: "POST"
+              });
+  issue_unassigned:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'unassigned'
+    steps:
+        - name: Set Data
+          id: set-unassigned-data
+          uses: actions/github-script@v6
+          env:
+            SECRET: ${{ secrets.AIRTABLE_KEY }}
+            WEBHOOK: ${{ secrets.AIRTABLE_WEBHOOK }}
+          with:
+            script: |
+              const body = JSON.stringify({
+                action: context.payload.action,
+                data: context.payload.issue.assignees,
+                owner: context.payload.repository.owner.login,
+                repo: context.payload.repository.name,
+                number: context.payload.issue.number.toString(),
+                secret: process.env.SECRET,
+                title: context.payload.issue.title,
+              });
+              return fetch(process.env.WEBHOOK, {
+                body,
+                headers: {
+                  "Content-Type": "application/json"
+                },
+                method: "POST"
+              });
+  issue_unlabeled:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'unlabeled'
+    steps:
+        - name: Set Data
+          id: set-unlabeled-data
+          uses: actions/github-script@v6
+          env:
+            SECRET: ${{ secrets.AIRTABLE_KEY }}
+            WEBHOOK: ${{ secrets.AIRTABLE_WEBHOOK }}
+          with:
+            script: |
+              const body = JSON.stringify({
+                action: context.payload.action,
```

Comment:
`context.payload` could be stored in a local var to DRY this up a bit. This applies to other step scripts.